Date and time picker docs (#757)
vivid-planet/comet: packages/admin/admin-stories/src/docs/components/DateAndTimePickers/02_TimePicker.stories.mdx

import { Meta, Canvas, Story } from "@storybook/addon-docs";

<Meta title="Docs/Components/Date & Time Pickers/Time Picker" />

# Time Picker

To use the `TimePicker` and `FinalFormTimePicker` components, install the `@comet/admin-date-time` package.

By default, there is only one selectable time option every 15 minutes. <br />
This shortens the list of options to reduce the need for scrolling. <br />
Use the `minuteStep` prop to change the step size.

> The range of selectable options can be limited by passing a time string into the `min` and `max` props.

Reviewer: Maybe add an example for a time string?

```suggestion
The range of selectable options can be limited by passing a time string (`HH:MM`) into the `min` and `max` props.
```

> ## Value type

Reviewer: Good idea to add a value type section!

> The `value` prop and the value returned by `onChange` are a 24-hour `string` formatted as `HH:MM`, similar to the [HTML time input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/time).

Reviewer: I guess it's either

```suggestion
The `value` prop and the value returned by `onChange` are 24-hour strings formatted as `HH:MM`, similar to the [HTML time input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/time).
```

or

```suggestion
The `value` prop and the value returned by `onChange` is a 24-hour `string` formatted as `HH:MM`, similar to the [HTML time input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/time).
```

## Basic usage

<Canvas>
    <Story id="stories-components-date-time-pickers-time-picker--basic" />
</Canvas>

> ## Usage inside a final-form

Reviewer: ```suggestion
## Usage inside a Final Form
```
